gtdynamics/universal_robot/Joint.h

```diff
@@ -286,15 +286,23 @@ class Joint : public boost::enable_shared_from_this<Joint> {
    * accel and a Values object containing this joint's angle Value and
    * derivatives.
    */
-  gtsam::Vector6 transformTwistAccelTo(
-      const LinkSharedPtr &link, double q, double q_dot, double q_ddot,
-      boost::optional<gtsam::Vector6> this_twist = boost::none,
+  gtsam::Vector6 transformTwistAccelTo1(
```

Comment:
Maybe have them disappear if not used, as lambdas.

Reply:
Done